.github/workflows/cherry-pick-next-to-master.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         uses: carloscastrojumo/github-cherry-pick-action@v1.0.9
         with:
           branch: master
-          token: ${{ secrets.GITHUB_TOKEN }}
           body: 'Cherry-pick of #{old_pull_request_id}'
+          cherry-pick-branch: cherry-pick-${old_pull_request_id}
```

Comment:
Currently, it would create a branch with name such name:
`cherry-pick-master-892c08f0604dccdac0ca2ab1221af88d17ccd5de`
WDYT about going with `cherry-pick-11446` instead? 🤔

Reply:
It [doesn't](https://github.com/MBilalShafi/test-github-action/pull/30) seem to work though.

Reply:
`old_pull_request_id` is only [supported](https://github.com/carloscastrojumo/github-cherry-pick-action/blob/3ea0e43fc7a86953536bd5faffef39c0507cd764/src/github-helper.ts#L57-L60) in the body (PR description)

Reply:
Thanks for confirming. 🙏 
Do you see any other option to reference the original PR in the branch name? 🤔

Reply:
This should work
```suggestion
          cherry-pick-branch: ${{ format('cherry-pick-{0}', github.event.number) }} 
```

Reply:
Nice! 👍 💙 
Looks great. 💯